MMM-NHL.js

```diff
@@ -217,15 +217,15 @@ Module.register('MMM-NHL', {
         this.nunjucksEnvironment().addFilter('formatStartDate', game => {
             const now = new Date();
             const inAWeek = now.setDate(now.getDate() + 7);
-            const start = new Date(game.timestamp);
+            const start = new Date(game.gameDate);
 
             if (start > inAWeek) {
-                return new Intl.DateTimeFormat(config.locale, {
+                return new Intl.DateTimeFormat(this.config.locale, {
```

Comment:
This is what I meant by the locale issue.  Not sure if its correct or not, but I made the change and added locale to the MMM-NHL config and now the dates display correctly.

Reply:
Ok that makes sense, I missed that in the docs.  Will revert.